awslimitchecker/tests/test_runner.py

```diff
@@ -614,6 +619,19 @@ def test_entry_no_service_name(self, capsys):
         assert excinfo.value.code == 6
         assert self.cls.service_name is None
 
+    def test_entry_no_service_name_skip_service(self, capsys):
```

Comment:
We can, but you've done more than enough work on this PR... that's entirely up to you. The test suite is pretty clunky, so I usually expect to add or rework some tests after merging PRs.